cannon/mipsevm/README.md

```diff
@@ -11,22 +11,44 @@ Supported 63 instructions:
 | `Logical`            | `andi`        | Bitwise AND immediate.                       |
```

Comment:
Another small change - line 3 says "Supported 63 instructions" - I think we can probably just cut that since it's annoying to keep up-to-date.

Reply:
There are also a few more references to `MIPS.sol` - in this doc and in cannon/docs/README.md